### DISCUSSION TITLE
I have table in which a single cell of each row have 3 different columns classified with white space how can I get that column different in each row list

### DISCUSSION THREAD

**Discussion by manish291740** [jsvine/pdfplumber#652] [Ask for help with specific PDFs]
Please describe, in as much detail as possible, your proposal and how it would improve your experience with pdfplumber.

**Comment by manish291740** [comment]


**Reply by jsvine** [reply]
Hi @manish291740, and thanks for your interest in `pdfplumber`.  Without having access to the specific PDF, it is difficult to provide the most helpful guidance. But, given the screenshots you show here, I'd recommend using the `"explicit_vertical_lines": [...],` table-extraction setting to designate the divisions between those columns. In this case, each item in that list would either be:

- An integer representing the `x` position of a full-height line. (The simplest approach.) Or:
- A `{ "x0": x, "top": top, "x1": x, "bottom": bottom }` dict, where `x` is the desired `x` position of the line, and `top` and `bottom` are the y-positions where you'd like the line to begin and end.
